dpctl-capi/include/dpctl_sycl_program_interface.h

```diff
@@ -56,10 +56,10 @@ DPPL_C_EXTERN_C_BEGIN
  * @return   A new SyclProgramRef pointer if the program creation succeeded,
  *           else returns NULL.
  */
-DPPL_API
-__dppl_give DPPLSyclProgramRef
-DPPLProgram_CreateFromOCLSpirv (__dppl_keep const DPPLSyclContextRef Ctx,
-                                __dppl_keep const void *IL,
+DPCTL_API
+__dpctl_give DPCTLSyclProgramRef
+DPCTLProgram_CreateFromOCLSpirv (__dpctl_keep const DPCTLSyclContextRef Ctx,
+                                __dpctl_keep const void *IL,
```

Comment:
leading white spaces. The issue exists in several places, may be it is a tabsize issue please take a look.